### ISSUE TITLE
JusticeMap tiles not working with E302

### ISSUE THREAD

**Issue by martinfleis** [leaflet-extras/leaflet-providers#495]
JusticeMap tiles do not seem to be working right now and trying to fetch individual tiles manually raises Error 302, indicating some form of a change. This note from the [website](http://www.justicemap.org/2020/) may be relevant

> Sept 19, 2021. If you are using our API, please contact us. Otherwise, as I haven't heard from anyone who is - the API is going to be revised and the old version will break (We might replace justicemap-api.org with another domain). Also if you have been using our tile layers with javascript - this might stop working (though you can easily switch to the newer 2020 layers, and we'll continue to support the 2010 layers just with a different url that you'll need to update).

I assume that we are pointing to the 2010 layer now that has moved, so this shall be updated (not sure how, what is within the scope of this project, if both 2010 and 2020 shall be listed etc.)

**Comment by brunob** [comment]
Linked to https://github.com/leaflet-extras/leaflet-providers/pull/224

Could you contact Aaron et point him to this issue ?

**Comment by martinfleis** [comment]
I dropped him an email to get his suggestions.

**Comment by martinfleis** [comment]
> Oops I forgot about this!
> 
> Hmm, we now have several different layers.  For race, we've got 2010 and 2020 US Census.
> 
> For income, we've got the latest five year ACS with the 2010 Census boundaries, and the ACS with the 2020 census boundaries.
> 
> 
> I put the 2020 data on a new server and never got a domain name for it (or I think for some reason the domain name didn't work):
> 2020 Race format
> http://198.245.60.70/tile/county/race/2020/plural/5/9/11.png
> 
> 2020 Income format
> http://198.245.60.70/tile/county/income/income/5/7/12.png
> 
> 2010 Income
> http://www.justicemap.org/tile/county/income/5/9/12.png
> 
> 2010 Race
> http://www.justicemap.org/tile/county/plural/5/8/13.png
> 
> Is that good or do you need more details?
> 
> Aaron

It is a question which layers shall be included here, given the project is apparently growing.

**Comment by brunob** [comment]
Thx for the feedback, i think we can include all layers, but i would need the complete list to put that in a PR.

**Comment by brunob** [comment]
any news about a complete list of layers available ?

**Comment by martinfleis** [comment]
I don't think it exists anywhere as an easy list to copy and Aaron hasn't sent it over.

In any case, the original reason why I opened this issue has resolved in the meantime.

**Comment by brunob** [comment]
So the bug is resolved and we can close this issue ?

**Comment by peterphalen** [comment]
I'm trying to resurrect a map involving the JusticeMap.income layer and it doesn't appear to work. Any thoughts on a potential fix? Thank you!
